documentation/source/library_reference.rst

```diff
@@ -195,6 +195,8 @@ Simulation Object Handles
 
 .. automodule:: cocotb.handle
     :members:
+    :private-members:
+    :exclude-members: _sub_handles, _invalid_sub_handles, _AssignmentResult, _SimIterator, _SetAction, _SetValueAction
```

Comment:
I'd prefer we write an exhaustive list of the members which _are_ public, rather than accidentally showing a private one

Reply:
Opened https://github.com/sphinx-doc/sphinx/pull/7473 to make this a little less painful.

Reply:
Thanks, I ran out of time fighting with ``autoattribute`` and ``autodata`` without success.

Reply:
That PR was merged - do we want to consider using an unreleased Sphinx from master?

Reply:
I wouldn't mind, if the pinning is not painful?